pkg/deprecation/deprecation.go

```diff
@@ -0,0 +1,16 @@
+package deprecation
```

Comment:
Can we put this under the pkg/util? I feel it is not big enough to be a separated package

Reply:
This has been moved